lib/bootloader_pvm.pm

```diff
@@ -162,13 +168,13 @@ sub prepare_pvm_installation {
         return (bootloader_pvm::prepare_pvm_installation $boot_attempt);
     }
 
-    assert_screen("run-yast-ssh", 300);
+    assert_screen("run-yast-ssh", 300) unless get_var('AGAMA_TEST');
 
     if (!is_upgrade && !get_var('KEEP_DISKS')) {
         prepare_disks;
     }
     # Switch to installation console (ssh or vnc)
-    select_console('installation');
+    select_console('installation') unless get_var('AGAMA_TEST');
```

Comment:
we are using `AGAMA: '1'` in s390x as shared variable indicating that is AGAMA, but only for the part that we are not creating ourselves, for qemu, we don't need it. Please apply this to all PR, instead of using AGAMA_TEST.

Reply:
We can't use `AGAMA: 1` because is not set in ppc64le, do you want to include it in the job group?

Reply:
yes, please.

Reply:
817b033f2